thirdparty/vars.sh

```diff
@@ -188,10 +188,10 @@ ROCKSDB_SOURCE=rocksdb-6.22.1
 ROCKSDB_MD5SUM="02727e52cdb94fa6a9dbbd68d157e619"
 
 # librdkafka
-LIBRDKAFKA_DOWNLOAD="https://github.com/edenhill/librdkafka/archive/v0.11.6-RC5.tar.gz"
-LIBRDKAFKA_NAME=librdkafka-0.11.6-RC5.tar.gz
-LIBRDKAFKA_SOURCE=librdkafka-0.11.6-RC5
-LIBRDKAFKA_MD5SUM="2e4ecef2df277e55a0144eb6d185e18a"
+LIBRDKAFKA_DOWNLOAD="https://github.com/edenhill/librdkafka/archive/v1.7.0.tar.gz"
```

Comment:
Are there any incompatibilities with higher versions?

Reply:
I have tested it and there is no compatibility issue